src/RoadRunnerGrpcApplicationRunner.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Runner\RoadRunner;
+
+use Spiral\RoadRunner\GRPC\Invoker;
+use Spiral\RoadRunner\GRPC\InvokerInterface;
+use Spiral\RoadRunner\GRPC\Server;
+use Spiral\RoadRunner\GRPC\ServiceInterface;
+use Spiral\RoadRunner\Worker;
+use Yiisoft\Yii\Runner\ApplicationRunner;
+
+/**
+ * `RoadRunnerGrpcApplicationRunner` runs the Yii gRPC application using RoadRunner.
+ */
+final class RoadRunnerGrpcApplicationRunner extends ApplicationRunner
+{
+    private ?InvokerInterface $invoker = null;
+    public bool $workerInterceptSideEffects = true;
```

Comment:
Should it be `public`? Can it be `withWorkerInterceptSideEffects()`? Should it be able to be changed runtime?

Reply:
I think it's completely redundant now. For the default worker, it is already set, and if you need to change it, then it is possible to use another worker - `withWorker`